UtopiaRDS/utopiaAirlineSchema.sql

```diff
@@ -19,17 +19,22 @@ CREATE TABLE `tbl_flights` (
   KEY `departure_idx` (`departure`),
   KEY `destination_idx` (`destination`),
   CONSTRAINT `departure` FOREIGN KEY (`departure`) REFERENCES `tbl_airports` (`code`),
-  CONSTRAINT `destination` FOREIGN KEY (`destination`) REFERENCES `tbl_airports` (`code`)
+  CONSTRAINT `destination` FOREIGN KEY (`destination`) REFERENCES `tbl_airports` (`code`),
+  CONSTRAINT `departure!=destination` CHECK ('departure' != 'destination'),

```

Comment:
Throughout, use backticks (`` ` ``), not single quotes (`'`), to quote column names; these constraints are asserting that, for each row, the string constants are not equal, are alphabetically ordered, and are or are not null.

Reply:
Good catch. They are changed in the newest pr